.circleci/config.yml

```diff
@@ -15,7 +15,7 @@ experimental:
 templates:
   job_template: &job_template
     docker:
-      - image: datadog/agent-buildimages-circleci-runner:v22284534-4d24e62
+      - image: datadog/agent-buildimages-circleci-runner_test_only:v23028262-7cab0fac
```

Comment:
For the record, beware when fixing the conflict here, the images are now pulled from GCR